### PR TITLE
[SPARK-34996][PYTHON] Port Koalas Series-related unit tests into PySpark

### PR DESCRIPTION
**Pull request by xinrong-meng**
### What changes were proposed in this pull request?
Now that we merged the Koalas main code into the PySpark code base (#32036), we should port the Koalas Series related unit tests to PySpark.

### Why are the changes needed?
Currently, the pandas-on-Spark modules are not fully tested. We should enable the Series related unit tests first.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
Enable Series-related unit tests.

Keyword:SPARK-34849

### CODE REVIEW

**Comment by xinrong-databricks** [comment]
CC @ueshin @HyukjinKwon @itholic

File: dev/sparktestsupport/modules.py [hunk 1]
```diff
@@ -612,6 +612,10 @@ def __hash__(self):
         "pyspark.pandas.typedef.typehints",
         # unittests
         "pyspark.pandas.tests.test_dataframe",
+        "pyspark.pandas.tests.test_series",
+        "pyspark.pandas.tests.test_series_datetime",
+        "pyspark.pandas.tests.test_series_string",
+        "pyspark.pandas.tests.test_series_conversion",
```
Comment (ueshin) [inline dev/sparktestsupport/modules.py:618]: We may want to list the tests in alphabetical order.

```
        "pyspark.pandas.tests.test_series_conversion",
        "pyspark.pandas.tests.test_series_datetime",
        "pyspark.pandas.tests.test_series_string",
```

Comment (xinrong-databricks) [reply dev/sparktestsupport/modules.py]: Good idea! Updated it.

**Review by ueshin — APPROVED** [review]
LGTM, pending tests.

**Review by itholic — APPROVED** [review]
LGTM if tests pass

**Comment by HyukjinKwon** [comment]
Merged to master.

**Pull request closed without merging**